quarkus/embedded/runtime/pom.xml

```diff
@@ -75,7 +75,7 @@
         <dependency>
             <groupId>org.jboss.narayana.jta</groupId>
             <artifactId>narayana-jta</artifactId>
-            <optional>true</optional>
+            <scope>compile</scope>
```

Comment:
Can just remove the line then?

Reply:
Without this I always get:

```java
[ERROR] /home/remerson/workspace/RedHat/infinispan/infinispan-alt/quarkus/embedded/runtime/src/main/java/org/infinispan/quarkus/embedded/runtime/graal/SubstituteTransactionClasses.java:[38,50] error: package com.arjuna.ats.jta does not exist
[ERROR] /home/remerson/workspace/RedHat/infinispan/infinispan-alt/quarkus/embedded/runtime/src/main/java/org/infinispan/quarkus/embedded/runtime/graal/SubstituteTransactionClasses.java:[46,33] error: package com.arjuna.ats.jta does not exist

```

Reply:
Ah, guess one of the poms must be setting the scope somewhere else :(